src/internal/models/ITranscodeActionModel.ts

```diff
@@ -0,0 +1,47 @@
+import {IActionModel} from "./IActionModel.js";
+
+interface IKeyframeIntervalsActionModel extends IActionModel{
+  interval?: number | string;
+}
+
+interface IFPSActionModel extends IActionModel{
+  fps?: number;
+}
+
+interface IFPSRangeActionModel extends IActionModel{
+  fps?: {from: number; to?: number}
+}
+
+interface IBitRateActionModel extends IActionModel {
+  bitRate?: number | string;
+  constant?: boolean;
+}
+
+interface IAudioCodecActionModel extends IActionModel{
+  audioCodec?: string;
```

Comment:
Please go over properties like this and check if they are required or not (: vs ?:)